src/org/testKitGen/MkGen.java

```diff
@@ -96,7 +96,21 @@ private void writeSingleTest(List<String> testsInPlaylist, TestInfo testInfo, Fi
 				f.write(testTargetName + ": TEST_GROUP=" + testInfo.getLevelStr() + "\n");
 				f.write(testTargetName + ": TEST_ITERATIONS=" + testInfo.getIterations() + "\n");
 				f.write(testTargetName + ": AOT_ITERATIONS=" + testInfo.getAotIterations() + "\n");
-
+				
+				// Set special openjdk problem list for JVM options that contains FIPS profile
+				// This feature is ignored if TEST_FLAG contains FIPS
+				if (arg.getBuildList().contains("openjdk")) {
+					if (!arg.getTestFlag().contains("FIPS") && !var.getJvmOptions().isEmpty() && var.getJvmOptions().contains("-Dsemeru.customprofile=")) {
```

Comment:
`var.getJvmOptions()` can be cached in a local variable.

Reply:
`-Dsemeru.customprofile=` can be assigned to a constant string for later multiple usages.

Reply:
updated